src/config/index.js

```diff
@@ -62,11 +63,18 @@ const config = new Map();
 
   config.set('CSP', CSP);
 
-  const APP_NAMES = ['search', 'disco'];
-  if (APP_NAMES.indexOf(process.env.APP_NAME) < 0) {
-    config.set('currentApp', 'core');
+  // This is the list of apps allowed to run.
+  const validAppNames = ['search', 'disco'];
+  config.set('validAppNames', validAppNames);
+
+  // Create a list of apps to build targets for.
+  const appName = process.env.APP_NAME;
+  if (validAppNames.indexOf(appName) > -1) {
```

Comment:
I had to revert that change as it's not in node. Will look at getting the polyfill working at some point but for quickness I reverted it.